tests/test_quantity.py

```diff
@@ -17,13 +17,30 @@
     IncompatibleValue,
     InsufficientArguments,
     NumPyRequired,
+    RequiresUniqueDimensions,
     get_si_value,
 )
 from ansys.units.unit import IncorrectUnits
 
 DELTA = 1.0e-5
 
 
+def test_preferred_units():
+    Quantity.preferred_units(units=["J", "slug", "psi"])
```

Comment:
As we've mentioned elsewhere, this feature is really useful if these choices are associated with quantity types. Given that, this seems to begin to intersect with the `api_quantity_map` feature that we're providing in YAML, but the way that we're using each piece of information is different at the moment.

Reply:
To be discussed further and handled in future work.